apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -236,9 +262,7 @@ init(StoreID) ->
 			?LOG_WARNING(WarningMessage);
 		false ->
 			ok
-	end,
-	{ok, #state{ file_index = FileIndex2, store_id = StoreID,
-			repack_cursor = read_repack_cursor(StoreID) }}.
```

Comment:
Oh do we only load the cursor from disk now if we're repacking in place? Not also if we're repacking across storage modules?